static/js/billing/billing.js

```diff
@@ -0,0 +1,60 @@
+var billing = (function () {
+var exports = {};
+
+exports.initialize = function () {
+    var stripe_key = $("#payment-method").data("key");
+    var handler = StripeCheckout.configure({ // eslint-disable-line no-undef
+        key: stripe_key,
+        image: '/static/images/logo/zulip-icon-128x128.png',
+        locale: 'auto',
+        token: function (token) {
+            var csrf_token = $("#payment-method").data("csrf");
+            $("#payment-section").hide();
+            $("#loading-section").show();
+            $.post({
+                url: "/json/billing/sources/change",
+                data: {
+                    stripe_token: JSON.stringify(token.id),
+                    csrfmiddlewaretoken: csrf_token,
+                },
+                success: function () {
+                    $("#loading-section").hide();
+                    $("#card-updated-message").show();
+                    setTimeout(function () {
+                        location.reload();
+                    }, 2000);
+                },
+                error: function (xhr) {
+                    $('#error-message-box').show().text(JSON.parse(xhr.responseText).msg);
+                },
+            });
+        },
+    });
+
+    $('#update-card-button').on('click', function (e) {
+        var email = $("#payment-method").data("email");
```

Comment:
What email is this getting? Ideally we wouldn't modify or display any email addresses at all.

Reply:
@rishig It's the current email of the logged in user sent from Zulip server. It would be displayed in the card update popup. If we don't provide a value for it Stripe would ask Customers to enter an email of their choice. I can't seem to figure out the use of this email though. My best guess is Stripe use the email as a parameter  in Radar for detecting frauds.

Reply:
hm, got it. Can we use `stripe_customer.email` for this instead, then?

Reply:
I think we can use that email. Stripe has not made it clear which email should be used here[1]. 
I have asked a question in StackOverflow regarding the use of email attribute. 
https://stackoverflow.com/questions/52217578/what-is-the-use-of-the-email-attribute-in-stripe-checkout-form/52223034#52223034

[1] https://stripe.com/docs/recipes/updating-customer-cards#adding-the-data-email-attribute

Reply:
I contacted Stripe customer care and they told to go with the email of the Customer object.